### PR TITLE
Updated README formatting, new Disqus section and various fixes

### PR DESCRIPTION
**Pull request by vegasbrianc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,11 +94,54 @@ var disqus_shortname = 'YOUR_DISQUS_SHORTCUT_HERE';
...
+## Enable Disqus or Facebook Comments
+This seciton will cover how to cover Disqus of Facebook commenting into the theme. Only enable either Disqus or Facebook comments.
+
+### Disqus Comments
+To enable Disqus comments update the code in Settings -> Code Injection -> Blog Footer.
+
+Insert your [Disqus shortname](https://shortname.disqus.com/admin/) in both the comments and Disqus comment count sections.
+
+To ensure the Disqus comment count is working correctly verify that the Disqus settings -> Comment & Community Configuration is set as seen below.
+![](./documentation/disqus_comment_count.png)
```
Comment (godofredoninja) [inline README.md]: @vegasbrianc  Please send the screenshot

Comment (vegasbrianc) [inline README.md]: Oops forgot the screenshot. It is added now.

**Comment by godofredoninja** [comment]
Hi @vegasbrianc Thank you for your collaboration in developing the theme for ghost 👍

**Comment by vegasbrianc** [comment]
Hi @godofredoninja are you working on the new version somewhere? Should I wait to fix the syntax?

**Comment by godofredoninja** [comment]
@vegasbrianc  Today I work on those small improvements and tomorrow I send it to the development branch [Dev](https://github.com/godofredoninja/Mapache/tree/dev)

**Pull request merged**